libs/portals/admin/regulations-admin/src/components/EditBasics.tsx

```diff
@@ -234,6 +237,38 @@ export const EditBasics = () => {
                   readOnly
                 />
               </Box>
+            ) : (
+              ministries.length > 0 &&
+              !draft.signatureText.value && (
+                <Select
+                  size="sm"
+                  label={t(m.regulationAdminMinistries)}
+                  name="setMinistry"
+                  isSearchable
+                  value={
+                    draft.ministry.value
+                      ? {
+                          value: draft.ministry.value,
+                          label: draft.ministry.value,
+                        }
+                      : undefined
+                  }
+                  placeholder={t(editorMsgs.selectMinistry)}
+                  options={[
+                    {
+                      label: t(editorMsgs.selectMinistry),
+                      value: undefined,
+                    },
+                    ...ministries.map((ministry) => ({
+                      value: ministry.name,
+                      label: ministry.name,
+                    })),
```

Comment:
No null checks needed on `ministry.name`?

Reply:
yeah good point, ill filter out the ones where no name is available, not sure it can happen but let's be safe here.